### PR TITLE
Feature/alert debug message

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/root/components/Alert/index.tsx [hunk 1]
```diff
@@ -68,21 +84,36 @@ function Alert<N extends string>(props: Props<N>) {
...
+            onClick={handleCopyDebugMessageButtonClick}
+            variant="transparent"
           >
-            <AiOutlineClose className={styles.icon} />
-          </RawButton>
-        )}
-      >
-        { children }
-      </ElementFragments>
+            Copy detailed error
```
Comment (tnagorra) [inline src/root/components/Alert/index.tsx:113]: No need to translate this?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/root/views/Country/ThreeW/NSProjects/SankeyFilters/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  const handleInputChange = React.useCallback((newValue: number[], name: string) => {
+    if (onChange) {
+      onChange((oldFilterValue) => {
+        const newFilterValue = {
+          ...oldFilterValue,
+          [name]: newValue,
+        };
+
+        return newFilterValue;
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/NSProjects/SankeyFilters/index.tsx:57]:
```suggestion
      onChange((oldFilterValue) => ({
          ...oldFilterValue,
          [name]: newValue,
        })
```

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 3]
```diff
@@ -23,7 +22,7 @@ const baseColumns = [
     'Total Budget',
     (item) => item.budget_amount,
     undefined,
-    { normal: true }
+    { normal: true, precision: 'auto' }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:25]:
```suggestion
    { normal: true, precision: 'auto' },
```

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 4]
```diff
@@ -40,14 +39,14 @@ const baseColumns = [
     'People Targeted',
     (item) => item.target_total,
     undefined,
-    { normal: true }
+    { normal: true, precision: 'auto' }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:42]:
```suggestion
    { normal: true, precision: 'auto' },
```

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 5]
```diff
@@ -40,14 +39,14 @@ const baseColumns = [
...
   ),
   createNumberColumn<Project, string | number>(
     'peopleReached',
     'People Reached',
     (item) => item.reached_total,
     undefined,
-    { normal: true }
+    { normal: true, precision: 'auto' }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:49]:
```suggestion
    { normal: true, precision: 'auto' },
```

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/root/views/Country/ThreeW/NSProjects/Map/index.tsx [hunk 6]
```diff
@@ -282,7 +372,19 @@ function ThreeWMap(props: Props) {
       navControlPosition="top-right"
       debug={false}
     >
-      <MapContainer className={_cs(styles.mapContainer, className)} />
+      <div className={_cs(styles.map, className)}>
+        <MapContainer className={styles.mapContainer} />
+        <div className={styles.legend}>
+          <LegendItem
+            color={COLOR_BLUE}
+            label="Reporting NS"
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/NSProjects/Map/index.tsx:380]: string left

File: src/root/views/Country/ThreeW/NSProjects/Map/index.tsx [hunk 7]
```diff
@@ -282,7 +372,19 @@ function ThreeWMap(props: Props) {
...
+          />
+          <LegendItem
+            color={COLOR_RED}
+            label="Receiving Country"
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/NSProjects/Map/index.tsx:384]: string left

File: src/root/views/Country/ThreeW/NSProjects/index.tsx [hunk 8]
```diff
@@ -105,12 +112,12 @@ function NSProjects(props: Props) {
     programmeTypeCounts,
     statusCounts,
   ] = React.useMemo(() => {
-    const ongoing = projectList.filter((p) => p.status === PROJECT_STATUS_ONGOING);
+    const ongoing = filteredProjectList.filter((p) => p.status === PROJECT_STATUS_ONGOING);
     const ongoingBudget = ongoing.reduce((acc, val) => acc + (+(val.budget_amount ?? 0)), 0);
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/NSProjects/index.tsx:116]: I think we created a sum function. Could you use it here?

File: src/root/views/Country/ThreeW/NSProjects/index.tsx [hunk 9]
```diff
@@ -105,12 +112,12 @@ function NSProjects(props: Props) {
...
-    const target = projectList.reduce((acc, val) => acc + (+(val.target_total ?? 0)), 0);
+    const target = filteredProjectList.reduce((acc, val) => acc + (+(val.target_total ?? 0)), 0);
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/NSProjects/index.tsx:117]: I think we created a sum function. Could you use it here?

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/root/views/Country/ThreeW/NSProjects/index.tsx (hunk 9 above)
Comment (tnagorra) [inline src/root/views/Country/ThreeW/NSProjects/index.tsx:117]: Let's use sum function please

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**